src/components/Banner/Banner.js

```diff
@@ -186,7 +186,7 @@ class Banner extends Component {
 
   render() {
     let locale = this.state.locale;
-    if (this.state.bandData?.active) {
```

Comment:
could you set that as a comment instead of deleting it? that way we can easily revert!